ci.py

```diff
@@ -167,6 +168,41 @@ def repo_cloned():
                 f.write(f'export {dep_name_underscore}_commit={commit_id}')
                 f.write(f'export {dep_name_underscore}_branch={dep["branch"]}\n')
 
+    def check_deps(self):
+        outdated_branches_message = 'Looks like there are outdated source branches.\n Please update them and restart ' \
+                                    'the job'
+        outdated_branches = []
+        submodules_url = '/'.join(PR_URL.split('/')[3:-2])
+        pull_number = PR_URL.split('/')[-1:][0]
```

Comment:
Please add example for CHANGE_URL here :)

Reply:
added above, thanks